frontend/src/main/java/cz/cuni/mff/xrg/odcs/frontend/gui/views/PipelineEdit.java

```diff
@@ -1178,6 +1188,9 @@ protected boolean savePipeline(final String successAction) {
         if (!validate()) {
             return false;
         }
+        if (!pipelineCanvas.validateGraph()) {
```

Comment:
Yes, it should be possible to save empty pipeline. But when running such pipeline better explanation of the failure should be provided